umake/frameworks/games.py

```diff
@@ -210,3 +210,42 @@ def post_install(self):
                         exec='"{}" %f'.format(self.exec_path),
                         comment=self.description,
                         categories="Development;IDE;"))
+
+
+class Superpowers(umake.frameworks.baseinstaller.BaseInstaller):
+
+    def __init__(self, category):
+        super().__init__(name="Superpowers", description=_("The HTML5 2D+3D game maker"),
+                         category=category, only_on_archs=['i386', 'amd64'],
+                         download_page="https://github.com/superpowers/superpowers/releases/latest",
+                         dir_to_decompress_in_tarball='superpowers*',
+                         desktop_filename="superpowers.desktop",
+                         required_files_path=["Superpowers"])
+
+    def parse_download_link(self, line, in_download):
+        """Parse Superpowers download links"""
+        url = None
+        if "-linux-" in line:
+            in_download = True
+
+        if in_download:
+            regexp = r'href="(.*-x64.zip)"'
```

Comment:
It's not an issue for now. I wouldn't make a default assumption on the architecture.
Say we needed in the future to add another architecture, this would download the x64 until we fixed it.
I'd do like the others and search for the architecture, or bail out.
It adds a variable (the self.bits in the eclipse installer), but to me it seems better.
Let me know what you and @didrocks think, it's just my opinion.